modules/oraclebackend/oraclebackend.hh

```diff
@@ -84,7 +84,7 @@ public:
   void setNotified(uint32_t zoneId, uint32_t serial) override;
   bool list(const DNSName& domain, int zoneId, bool include_disabled=false) override;
   bool startTransaction(const DNSName& domain, int zoneId) override;
-  bool feedRecord(const DNSResourceRecord &rr, const DNSName ordername) override;
+  bool feedRecord(const DNSResourceRecord &rr, const DNSName ordername);
```

Comment:
This doesn't seem right, contrary to the two other ones `feedRecord()` is really inherited and should match the declaration of the base class, unless I'm missing something.